### PR TITLE
Stocks prediction demo

### PR DESCRIPTION
**Pull request by daniels290813**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,3 +128,22 @@ The code includes the following components:
 
 <p><img src="./docs/converting-to-mlrun-pipeline.png" alt="converting-to-mlrun pipeline output"/></p>
 
+<a id="stock-prediction"></a>
+## Stocks prices prediction demo
+
+The [**stock-prediction**](stock-prediction/README.md) demo demonstrates how to use feature store to create two ingestion services - news and stocks, news pipeline enriches the stocks news scraped from [yahoo-api](https://finance.yahoo.com/) with its sentiments using a pretrained sentiment analysis model.<br>
+It uses MLRun's pytoch framework to train RNN based model on a feature-vector and creates a model serving function with a custom predict method.<br>
```
Comment (jillnogold) [inline README.md:135]: pytorch

File: README.md [hunk 2]
```diff
@@ -127,3 +128,22 @@ The code includes the following components:
 
 <p><img src="./docs/converting-to-mlrun-pipeline.png" alt="converting-to-mlrun pipeline output"/></p>
 
+<a id="stock-prediction"></a>
+## Stocks prices prediction demo
+
+The [**stock-prediction**](stock-prediction/README.md) demo demonstrates how to use feature store to create two ingestion services - news and stocks, news pipeline enriches the stocks news scraped from [yahoo-api](https://finance.yahoo.com/) with its sentiments using a pretrained sentiment analysis model.<br>
```
Comment (jillnogold) [inline README.md:134]: The [**stock-prediction**](stock-prediction/README.md) demo demonstrates how to use feature store to create two ingestion services - news and stocks. News....

**Comment by daniels290813** [comment]
@jillnogold Thanks alot for you feedback

**Comment by aviaIguazio** [comment]
@daniels290813 if graphana notebook is not valid please remove it

**Comment by aviaIguazio** [comment]
postponed to 1.2.0

**Pull request closed without merging**